src/rgw/rgw_common.cc

```diff
@@ -201,7 +201,7 @@ static string get_abs_path(const string& request_uri) {
   return request_uri.substr(beg_pos, len - beg_pos);
 }
 
-req_info::req_info(CephContext *cct, class RGWEnv *e) : env(e) {
+req_info::req_info(CephContext *cct, class RGWEnv *_env) : env(_env) {
```

Comment:
Yeah I thought of going that route, and got around to fixing all the places where we mutate the map without the methods. I also vote for this.